snippets/web/functions-code.snippets.code-snippets

```diff
@@ -1,40 +1,28 @@
 {
-
-	"create Function" :{
-		"scope": "javascript , typescript",
-		"prefix": "createFunc",
-		"body": ["const ${1:response} = await ${2:function}.create(`${3:[FUNCTION_ID]}` , `${4:[name]}`)"] ,
```

Comment:
We need this snippet but the body is incorrect.

What I was expecting here is to be 
`const ${1:function} =  Function(${2:client});`

and in the description, it must be. "Create a new Function object and initialize it"